### DISCUSSION TITLE
running both a tunnel to a Nodejs server over https *and* let me ssh into the machine.

### DISCUSSION THREAD

**Discussion by jab-r** [ngrok/ngrok#44] [Q&A]
this should be a FAQ but I'm a newbie and can't find a braindead simple answer.
 
 I'm developing on Ubuntu and my ngrok.yml is very simple (contains my authtoken)

I've also installed ngrok as a systemd service with the relevant lines:

WorkingDirectory=/home/jon
ExecStart=/snap/bin/ngrok tunnel --label edge=edg... http://localhost:80

so I understand that I need to create a .yml with the endpoints but honestly I don't know what to put in the endpoints, I do have ngrok endpoints for both my reverse-http proxy, and created another for SSH

I also created a separate ngrok-ssh service but I can't start both. I think I need really basic instructions

**Comment by russorat** [comment]
@jab-r thanks for the discussion, i'm so sorry i missed this. Are you still having issues with this? Did you ever figure it out?

**Comment by jab-r** [comment]
Thanks, and yes its working now!

**Reply by russorat** [reply]
great to hear!
